javascript/index.js

```diff
@@ -31,35 +41,84 @@ function printMilliseconds() {
 }
 
 function printSplit() {
+  splitsElement.innerHTML += '<li>' + minDecElement.innerHTML + minUniElement.innerHTML+ ':'+ secDecElement.innerHTML + secUniElement.innerHTML + '</li>';
   // ... your code goes here
 }
```

Comment:
wenn du Interesse hast, guck nochmal in die Lösungen, die wir hochladen, es gibt noch andere Möglichkeiten die Aufgabe zu lösen